test/opt/ir_builder.cpp

```diff
@@ -296,6 +296,96 @@ OpFunctionEnd
   Match(text, context.get());
 }
 
+TEST_F(IRBuilderTest, ConstantAdder) {
+  const std::string text = R"(
+; CHECK: [[uint:%\w+]] = OpTypeInt 32 0
+; CHECK: [[u0:%\w+]] = OpConstant [[uint]] 13
```

Comment:
If you need to match a captured variable later, I would suggest not capturing it. Instead you could have:
```
; CHECK: [uint:%\w+]] = OpTypeInt 32 0
; CHECK: OpConstant [[uint]] 13
; CHECK: [[sint:%\w+]] = OpTypeInt 32 1
; CHECK: OpConstant [[sint]] -1
...
```